haskell-style.md

```diff
@@ -280,13 +281,18 @@ Sort your exports, unless the order matters in your desired Haddock output.
     - TypeFamilies
   ```
 
-  **NOTE**: `NoImplicitPrelude` may be omitted in packages using the normal,
-  implicit `Prelude` everywhere.
-
   This defines a consistent, and minimally-extended Haskell environment. Other
   extensions MUST be defined via LANGUAGE pragmas in the modules where they're
   needed.
 
+  We allow our `entities` package to diverge from this list, since it is almost
```

Comment:
Kinda wish this wasn't true, but better to document than not.

Reply:
I'm open to making ourselves put all the pragmas in all the modules, if you wanna push on it.

TH/QQ are already explicitly in all the modules right now, but we elected to make `UndecidableInstances` special when a persistent update introduced its need. I'm purely working the consistency angle here, but moving TH/QQ int he direction of UI. If we worked the consistency angle the other way, and forced moving UI to a pragma in all the modules instead, I'd be perfectly happy.

Reply:
`entities` definitely feels special so this isn't wildly problematic for me; it makes sense and I doubt it will be questioned frequently, if at all